.travis.yml

```diff
@@ -149,6 +151,10 @@ jobs:
       after_failure: post_message "TRAVIS_MESSAGE" "'make dist' failed"
 
 
+    - name: Re-check Shell Scriopts (after build) with shellcheck
```

Comment:
There is a typo problem here `Scriopts`, that does not affect the main goal of the PR, so I will approve it, and case you change this, I will approve again.

Reply:
Fixed 🤣